audio/softsynth/mt32/Analog.cpp

```diff
@@ -223,23 +223,23 @@ class AnalogImpl : public Analog {
 		delete &rightChannelLPF;
 	}
 
-	unsigned int getOutputSampleRate() const override {
+	unsigned int getOutputSampleRate() const {
 		return leftChannelLPF.getOutputSampleRate();
 	}
 
-	Bit32u getDACStreamsLength(const Bit32u outputLength) const override {
+	Bit32u getDACStreamsLength(const Bit32u outputLength) const {
 		return leftChannelLPF.estimateInSampleCount(outputLength);
 	}
 
-	void setSynthOutputGain(const float synthGain) override;
-	void setReverbOutputGain(const float reverbGain, const bool mt32ReverbCompatibilityMode) override;
+	void setSynthOutputGain(const float synthGain);
+	void setReverbOutputGain(const float reverbGain, const bool mt32ReverbCompatibilityMode);
 
-	bool process(IntSample *outStream, const IntSample *nonReverbLeft, const IntSample *nonReverbRight, const IntSample *reverbDryLeft, const IntSample *reverbDryRight, const IntSample *reverbWetLeft, const IntSample *reverbWetRight, Bit32u outLength) override;
-	bool process(FloatSample *outStream, const FloatSample *nonReverbLeft, const FloatSample *nonReverbRight, const FloatSample *reverbDryLeft, const FloatSample *reverbDryRight, const FloatSample *reverbWetLeft, const FloatSample *reverbWetRight, Bit32u outLength) override;
+	bool process(IntSample *outStream, const IntSample *nonReverbLeft, const IntSample *nonReverbRight, const IntSample *reverbDryLeft, const IntSample *reverbDryRight, const IntSample *reverbWetLeft, const IntSample *reverbWetRight, Bit32u outLength);
+	bool process(FloatSample *outStream, const FloatSample *nonReverbLeft, const FloatSample *nonReverbRight, const FloatSample *reverbDryLeft, const FloatSample *reverbDryRight, const FloatSample *reverbWetLeft, const FloatSample *reverbWetRight, Bit32u outLength);
 
 	template <class Sample>
 	void produceOutput(Sample *outStream, const Sample *nonReverbLeft, const Sample *nonReverbRight, const Sample *reverbDryLeft, const Sample *reverbDryRight, const Sample *reverbWetLeft, const Sample *reverbWetRight, Bit32u outLength) {
-		if (outStream == nullptr) {
+		if (outStream == NULL) {
```

Comment:
Please don't replace nullptr (which is C++ style) with NULL (which is C style).

Check here why NULL was replaced with nullptr:
https://stackoverflow.com/questions/20509734/null-vs-nullptr-why-was-it-replaced

Reply:
I didn't do the modifications on my own - I took the code directly from mt32emu (as always)...

Reply:
I wonder what is the better approach here: modify the sources we import (and deviate from the original) or keeping the imported code as-is, "trusting" the original developers? I don't think they made this change without a reason though...

Reply:
Looks like upstreams always used NULL, and we changed it in https://github.com/scummvm/scummvm/commit/af529f568baa1f9e47a957e84fd17561c15613e0